src/pages/AddVacation/AddVacation.tsx

```diff
@@ -18,12 +18,12 @@ const AddVacation = () => {
   const [endDate, setEndDate] = useState<Dayjs | null>(null);
   const [notes, setNotes] = useState<string>("");
   const navigate = useNavigate();
-  const { data, setData } = useContext(UserData);
+  const { data, setData, setFilterType } = useContext(UserData);
   const { user } = useAuth();
   const { vacationId } = useParams();
 
   const isFieldsEmpty = startDate && endDate && vacationType;
-  const setHeaderTitle: string = !vacationId ? "Add vacation" : "Edit vacation";
+  const setHeaderTitle: string = vacationId ? "Edit vacation" : "Add vacation";
```

Comment:
Don't forget that according to the requirements in case of editing title should also show the ID of the entity that is currently editing.

Example: "Edit vacation #12"